packages/cardano-services/src/ChainHistory/DbSyncChainHistory/mappers.ts

```diff
@@ -148,16 +170,17 @@ export const mapCertificate = (
   if (isStakeCertModel(certModel))
     return {
       __typename: certModel.registration
-        ? Cardano.CertificateType.StakeRegistration
-        : Cardano.CertificateType.StakeDeregistration,
+        ? Cardano.CertificateType.Registration
+        : Cardano.CertificateType.Unregistration,
       cert_index: certModel.cert_index,
+      deposit: BigInt(certModel.deposit),
```

Comment:
Is this available in the db? It's not part of the old certificates

Reply:
Yes it is, not directly in the `stake_registration` table (some `JOIN`s with `epoch_parameter` table is required) and this is common both to old and new certificates.
What's missing in the **db-sync** schema is the ability to distinguish a new certificate from an old certificate.
Anyway the old certificate is a subset of the new certificate and they are equivalent (at logical level there is no changes regardless from the certificate has or not an explicit `deposit` attribute in the tx body) so we map both of them to the same Core type.